databases/week2/homework2.sql

```diff
@@ -0,0 +1,108 @@
+--  Part one
+--  Working with tasks
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+
+INSERT INTO task (title, description, created, updated, due_date, status_id, user_id)
+VALUES ('Finishing the home work','edit current program','2020-04-12','2020-04-13','2020-09-21',3,6);
+
+-- Change the title of a task
+
+UPDATE task
+SET title = 'Remove previous week HW'
+WHERE title ='Finishing the home work';
+
+
+-- Change a task due date
+
+UPDATE task
+SET due_date = '2020-10-01'
+WHERE title ='Remove previous week HW';
+
+--  Change a task status
+
+UPDATE task
+SET status_id = 1
+WHERE id = 35;
+
+-- Mark a task as complete
+
+UPDATE task
+SET status_id = 3
+WHERE title='Wash clothes';
```

Comment:
in case of duplicate title, will this update query work?

Reply:
Hey Abed, Does it means I should take that task only with the id to fulfill this section of exercise?